src/EnergyPlus/FluidProperties.cc

```diff
@@ -2423,9 +2418,7 @@ namespace FluidProperties {
         Real64 Temperature; // Temperature to drive values
         Real64 ReturnValue; // Values returned from refrigerant functions
 
-        auto &df = state.dataFluidProps;
-
-        for (auto *refrig : df->refrigs) {
+        for (auto *refrig : state.dataFluidProps->refrigs) {
```

Comment:
[src/EnergyPlus/FluidProperties.cc:2426]:(style),[constVariable],Variable 'df' can be declared as reference to const